lib/ansible/modules/apt.py

```diff
@@ -135,6 +135,14 @@
     type: bool
     default: 'no'
     version_added: "2.1"
+  fail_on_autoremove:
+    description:
+      - 'Corresponds to the C(--no-remove) option for I(apt).'
+      - 'If C(yes), it is ensured that no packages will be removed or the task will fail.'
+      - 'C(fail_on_autoremove) is only supported with state except I(absent)'
```

Comment:
`I()` formats as italic. These should receive inline code formatting.
```suggestion
      - 'Corresponds to the C(--no-remove) option for C(apt).'
      - 'If C(yes), it is ensured that no packages will be removed or the task will fail.'
      - 'C(fail_on_autoremove) is only supported with state except C(absent)'
```